backend/services/core/src/schema.graphql

```diff
@@ -196,6 +200,9 @@ input CreatePosition {
   id: String!
   name: String!
   committee_id: UUID
+  email: String
```

Comment:
Shouldn't email be required?

Reply:
Some positions especially old ones do not have an email.